src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DeprecationControl.xaml

```diff
@@ -50,12 +58,18 @@
           <StackPanel
             Orientation="Horizontal"
             VerticalAlignment="Center">
-            <TextBox
-              Style="{DynamicResource SelectableTextBlockStyle}" 
+            <!-- Style="{DynamicResource SelectableTextBlockStyle}" -->
```

Comment:
Any reason why we shouldn't remove this?

Reply:
I don't want to affect current style. That's why I left there. May need to remove it later.

Reply:
Not sure what you mean by current style. 

It's code that's not relevant anymore, we should remove it.

Reply:
Removed.